### ISSUE TITLE
Strange Project Tree Behavior with "Set As Default Application"

### ISSUE THREAD

**Issue by JoelProminic** [prominic/Moonshine-IDE#332]
I noticed this behavior on macOS while reviewing #306:
1. Create a new Flex Browser project
2. Create a new MXML file in src, based on Application (i.e. AdditionalApplication.mxml)
3. Expand the bin-debug folder in the project tree
4. Right-click on the new MXML file and click "Set as Default Application".  BUG.  This generates another bin-debug directory underneath the current bin-debug directory.
5. Right-click on the original MXML file and click "Set as Default Application".  BUG.  This generates another bin-debug directory underneath the current bin-debug directory.
6. Refresh the bin-debug directory.  The duplicate bin-debug directories disappear from the tree.

I see a related behavior for a desktop project
1. Create a new Flex Desktop project
2. Create a new MXML file in src, based on Application (i.e. AdditionalApplication.mxml)
3. Right-click on the new MXML file and click "Set as Default Application"
4. Right-click on the original MXML file and click "Set as Default Application".  BUG: this creates a duplicate of the original -app.xml file.
5. Refresh the src directory.  The duplicate -app.xml file disappears from the tree.

The other project types may have similar behavior, but I did not test them today.

This issue is mostly cosmetic, so it is not a high priority.

**Comment by rat-moonshine** [comment]
The above issue should be fixed now across web and desktop applications. Please, check. 

We shall need to include this in CHANGELOG.md, as well.

**Comment by JoelProminic** [comment]
I did tested the other projects to see if they had similar problems.   I tested the above logic only - I did not try to verify that the default application file was switched properly.

Project Template | Behavior
---------------- | --------
ActionScript Desktop | Good
ActionScript Web | Good
Away3D | Good
Feathers Desktop | Good
Feathers Mobile | Good
Flex Desktop | Good
Flex Web | Good
Flex Mobile | Good
Flex Library | N/A
Royale | Good
Visual Editor | N/A

It looks like the fix worked for all of the applications.
